shared/middleware/helpers/storage_apps.rb

```diff
@@ -62,6 +62,29 @@ def update(channel_id, value, ip_address)
     JSON.parse(row[:value]).merge(id: channel_id, isOwner: owner == @storage_id, updatedAt: row[:updated_at])
   end
 
+  def publish(channel_id, type)
+    owner, id = storage_decrypt_channel_id(channel_id)
+    raise NotFound, "channel `#{channel_id}` not found in your storage" unless owner == @storage_id
+    row = {
+      project_type: type,
```

Comment:
We have to set `project_type` here, because it's not guaranteed to be set for existing projects.